### PR TITLE
Cb 26

### PR DESCRIPTION
**Pull request by CalebMacdonaldBlack**

### CODE REVIEW

File: test/clj/centrebull/test/db/ranges.clj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(ns centrebull.test.db.ranges
+  (:require [clojure.test :refer :all]))
+
+(defn create!
+  [em rv]
+  (fn [range]
+    (is (= range em))
```
Comment (axrs) [inline test/clj/centrebull/test/db/ranges.clj]: Swap the order of these values for comparision.

They typical order in clojure (to have it reflect correctly in intelliJ ) on error is : expected then actual

File: test/clj/centrebull/test/mock_generators.clj [hunk 2]
```diff
@@ -19,6 +19,11 @@
          :shooter/preferred-name (last-name)
          :shooter/club           (string)}))
 
+(defn gen-range
+  ([] (gen-range (rand-int 99999)))
```
Comment (axrs) [inline test/clj/centrebull/test/mock_generators.clj]: This should be generating a uuid. Have a look at the competition generator (may require merging develop into this branch to get the function)

File: test/clj/centrebull/test/db/ranges.clj (hunk 1 above)
Comment (axrs) [reply test/clj/centrebull/test/db/ranges.clj]: It's not a major thing, but it's something I'm working to enforce where I see it. (and trying to use myself).

**Review by axrs — CHANGES_REQUESTED** [review]

**Comment by CalebMacdonaldBlack** [comment]
@axrs

**Review by axrs — APPROVED** [review]

**Pull request merged**